libraries/AP_HAL_ChibiOS/hwdef/MicoAir743-AIO/hwdef-bl.dat

```diff
@@ -0,0 +1,47 @@
+# hw definition file for processing by chibios_hwdef.py
```

Comment:
you might ocnsider adding SD card interface in the bootloader to allow updating firmware directly from the SD card...not a blocker though

Reply:
Okay! I will test and update the config files later.

Reply:
Hi! I believe updating firmware from the SD card is not convenient, so I didn't include that option in the bootloader.